src/resolvers/draftDonationResolver.ts

```diff
@@ -56,6 +56,10 @@ export class DraftDonationResolver {
     useDonationBox?: boolean,
     @Arg('relevantDonationTxHash', { nullable: true })
     relevantDonationTxHash?: string,
+    @Arg('toWalletMemo', { nullable: true }) toWalletMemo?: string,
+    @Arg('qrCodeDataUrl', { nullable: true }) qrCodeDataUrl?: string,
+    @Arg('isQRDonation', { nullable: true, defaultValue: false })
+    isQRDonation?: boolean,
```

Comment:
Please add more test cases for this endpoint to cover new conditions like passing these enw params